.github/workflows/stale-issue-cleanup.yml

```diff
@@ -2,11 +2,10 @@ name: "Close stale issues"
 
 # Controls when the action will run.
 on:
-  # allows doing a manual trigger of the action
+  # allows doing a manual trigger of the action and running it on a schedule at midnight everyday
```

Comment:
What timezone is the cron schedule anchored to?

Reply:
It is anchored to UTC. I will add a comment and update the schedule to run at 10am PT